guest-agents/vars.yaml

```diff
@@ -8,3 +8,5 @@ PCRE2_VERSION: 10.42
 XEN_GUEST_AGENT_VERSION: 0.4.0
 # renovate: datasource=github-releases depName=siderolabs/talos-vmtoolsd
 TALOS_VMTOOLSD_VERSION: v0.6.0
+# renovate: datasource=github-releases depName=siderolabs/talos-metal-agent
+TALOS_METAL_AGENT_VERSION: v0.1.0
```

Comment:
Will work only after we merge https://github.com/siderolabs/talos-metal-agent and release this version as the initial version.